spec/models/concerns/wrangleable_spec.rb

```diff
@@ -10,23 +10,30 @@
 
   describe "#update_last_wrangling_activity" do
     context "as a tag wrangler" do
-      before { User.current_user = create(:tag_wrangler) }
+      before do
+        User.current_user = create(:tag_wrangler)
+        User.current_user.last_wrangling_activity.updated_at = 60.days.ago
+        User.current_user.last_wrangling_activity.save!(touch: false)
+      end
 
       context "a wrangling activity has happened" do
         before { User.should_update_wrangling_activity = true }
 
         it "sets a last wrangling time" do
           freeze_time do
             expect(wrangleable.save).to be_truthy
-            expect(User.current_user.last_wrangling_activity.updated_at).to eq(Time.now.utc)
+            expect(User.current_user.reload.last_wrangling_activity.updated_at).to eq(Time.now.utc)
```

Comment:
Should this be `Time.current`?

Reply:
Yes. I'll batch it with other review changes if there are any.